modules/efk-logging-elasticsearch-add-remove.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
```

Comment:
@ewolinetz Looking at this section in detail, I have reservations about describing, in detail, how to modify your ES cluster as it all needs to occur in "unmanaged' mode.  How far down this rabbit hole do we really want to go?